backend/app/tests/conftest.py

```diff
@@ -242,3 +243,11 @@ def exists_side_effect(self):
         mock_exists.side_effect = exists_side_effect
 
     return _mock_exists
+
+
+@pytest.fixture
+def mock_session():
+    with patch("app.db.database.SessionLocal") as mock_session_local:
+        mock_session = create_autospec(Session, instance=True)
+        mock_session_local.return_value = mock_session
+        yield mock_session
```

Comment:
When I run `make test` the test user is added to the database. The user is never removed.